### PR TITLE
Allow controlling frequency

### PR DESCRIPTION
**Pull request by moshez**

### CODE REVIEW

**Comment by moshez** [comment]
Addresses #8

File: txppb/__init__.py [hunk 1]
```diff
@@ -14,7 +14,8 @@ class _FinishLoop(Exception):
 @defer.inlineCallbacks
 def main_loop(
         reactor: interfaces.IReactorTime,
-        engine: ppb.engine.GameEngine
+        engine: ppb.engine.GameEngine,
+        frequency: ppb.engine.GameEngine=None,
```
Comment (pathunstrom) [inline txppb/__init__.py]: Pretty sure this is supposed to be int or float?

**Review by pathunstrom — CHANGES_REQUESTED** [review]

File: txppb/__init__.py (hunk 1 above)
Comment (moshez) [reply txppb/__init__.py]: ugh, egg on my face. of course. Fixed!

**Review by pathunstrom — APPROVED** [review]
Looks good.

**Pull request merged**